packages/components/components/elvis-modal/src/react/elvia-modal.tsx

```diff
@@ -66,16 +66,24 @@ export const Modal: FC<ModalProps> = function ({
   const hasSecondaryButton = !!secondaryButton || !!webcomponent?.getSlot('secondaryButton');
   const hasHeading = !!heading || !!webcomponent?.getSlot('heading');
 
+  const lang = useLanguage();
+
   const getAriaLabel = (): string => {
     if (heading && typeof heading === 'string') {
       return heading;
     } else if (webcomponent?.getSlot('heading')) {
-      return webcomponent.getSlot('heading').textContent ?? 'Forgrunnsvindu';
+      return (
+        webcomponent.getSlot('heading').textContent ?? (lang === 'no' ? 'Forgrunnsvindu' : 'Popup window')
+      );
     } else {
-      return 'Forgrunnsvindu';
+      return lang === 'no' ? 'Forgrunnsvindu' : 'Popup window';
     }
   };
 
+  const getCloseAriaLabel = (): string => {
```

Comment:
```suggestion
  const getCloseAriaLabel = () => {
```
If you remove the explicit return type of this function here, you can let TypeScript infer the return type. When you hover over the function it will show that it can only return "`'Forgrunnsvindu'`" or "`'Popup window'`", making it clearer.


before:
![Skjermbilde 2024-09-24 kl  08 47 30](https://github.com/user-attachments/assets/85d5aa91-66bd-42af-bdd1-b7c51cde8f83)

after:
![Skjermbilde 2024-09-24 kl  08 47 42](https://github.com/user-attachments/assets/5a918cd7-bdc7-4445-b1f5-bcaac2a85ca0)

Reply:
I didn't think of that. I've made the changes that you've suggested to make it easier for others to see what the function returns.